demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1907,45 +2005,57 @@ export class DemoMeetingApp
           this.appendNewSpeakerTranscriptDiv(segment, speakerToTranscriptSpanMap);
         } else {
           const transcriptSpan = speakerToTranscriptSpanMap.get(newSpeakerId);
-          transcriptSpan.innerText = transcriptSpan.innerText + '\u00a0' + segment.content;
+          transcriptSpan.innerText = transcriptSpan.innerText + '\u00a0';
+          transcriptSpan.appendChild(segment.contentSpan);
         }
       }
     }
   };
 
   populatePartialTranscriptSegmentsFromResult = (segments: TranscriptSegment[], result: TranscriptResult) => {
     let startTimeMs: number = null;
-    let content = '';
     let attendee: Attendee = null;
+    let contentSpan;
     for (const item of result.alternatives[0].items) {
+      const itemContentSpan = document.createElement('span') as HTMLSpanElement;
+      itemContentSpan.innerText = item.content;
+      itemContentSpan.classList.add('transcript-content');
+      if (item.hasOwnProperty('confidence') && !item.content.startsWith("[") && item.confidence < 0.5) {
+        itemContentSpan.classList.add('confidence-style');
+      }
+      if (this.transcriptEntitySet.size > 0 && this.transcriptEntitySet.has(item.content)) {
+        itemContentSpan.classList.add('entity-color');
+      }
```

Comment:
This would really benefit from some examples to show what `item` might be in each case.